### PR TITLE
Use san plot to determine the noise level of 1D spectrum.

### PR DESCRIPTION
**Pull request by jobo322**

### CODE REVIEW

**Comment by lpatiny** [comment]
Could you check the testcases

File: package.json [hunk 1]
```diff
@@ -75,5 +77,8 @@
     "node-fetch": "^2.6.1",
     "openchemlib-utils": "^1.0.0",
     "spectrum-generator": "^4.7.1"
+  },
+  "volta": {
+    "node": "16.0.0"
```
Comment (targos) [inline package.json:82]: it shouldn't be necessary to lock the version of node.js on our libraries

Comment (jobo322) [reply package.json:82]: oh yes, it was a mistake from my side. Do you know if there are problems having nvm and volta installed?. I usually use npm link to test locally and I found that I need to match the versions of projects in order to get working the linking

Comment (targos) [reply package.json:82]: I don't know, I only have volta.
You can temporarily change the node version if needed (similar to `nvm use 16` for example) using: `volta run --node=16 bash`

**Pull request merged**